File: Tellma/ClientApp/src/app/data/dto/export-arguments.ts
// tslint:disable:variable-name
import { GetArguments } from './get-arguments';

export interface ExportArguments extends GetArguments {
  format?: 'csv' | 'xlsx';
}

export const ExportArguments_Format = {
  xlsx: 'Excel',
  csv: 'CSV'
};

File: Tellma/Controllers/DTO/ExportArguments.cs
using Tellma.Controllers.Utilities;
using Tellma.Entities;

namespace Tellma.Controllers.Dto
{
    public class ExportArguments : GetArguments
    {
        // Same parameter exists in TemplateArguments
        [ChoiceList(new object[] { FileFormats.Xlsx, FileFormats.Csv })]
        public string Format { get; set; } = FileFormats.Xlsx;
    }
}
